internal/webhooks/inclusterippool.go

```diff
@@ -147,3 +156,50 @@ func (webhook *InClusterIPPool) validate(_, newPool *v1alpha1.InClusterIPPool) (
 
 	return //nolint:nakedret
 }
+
+func validateAddresses(newPool *v1alpha1.InClusterIPPool) field.ErrorList {
+	var allErrs field.ErrorList
+
+	if newPool.Spec.Subnet != "" {
+		allErrs = append(allErrs, field.Invalid(field.NewPath("spec", "subnet"), newPool.Spec.Subnet, "subnet may not be used with addresses"))
```

Comment:
not too sure, but it might be nice to allow using subnet. If we allow it, we should also default it to the correct subnet according to addresses and prefix.

Reply:
Valid point, let's keep it this way.